drug-drug-interaction/README.md

```diff
@@ -0,0 +1,41 @@
+### Drug-Drug Interaction
+
+## Problem
+In this example, we attempt to infer unknown drug drug interactions from a network of
+multiple drug-based similarities and known interactions. 
+
+## Dataset
```

Comment:
This sections needs a short introductory paragraph before launching into each dataset.
Maybe add a sentence about there being three different datasets derived from DrugBank (include a link), and then move the last paragraph in this section into this one.